src/openrct2-ui/UiStringIds.h

```diff
@@ -1152,12 +1152,13 @@ namespace OpenRCT2
         STR_OPTIONS_ADVANCED = 6167,
         STR_OPTIONS_AUDIO_TIP = 5268,
         STR_OPTIONS_AUTOSAVE_FREQUENCY_LABEL = 2700,
-        STR_OPTIONS_CONTROLS_AND_INTERFACE_TIP = 5269,
+        STR_OPTIONS_CONTROLS_TIP = 5269,
```

Comment:
To do: looks like the actual modified 'controls' string got lost along the way. I'll have to re-modify it.